src-tauri/src/main.rs

```diff
@@ -474,6 +474,7 @@ pub enum InstallType {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct GameInstall {
     pub game_path: String,
+    pub launch_parameters: String,
```

Comment:
adding it to GameInstall is not ideal, but the alternative is passing it to the function directly, which would change its signature.
Consider this a stop-gap implementation

Reply:
You can make function signature use `Option<String>` for launch args. That changes the signature on Rust side but doesn't require any change on UI side, as missing function arg on ui is counted as `None` when sent to backend ^^

Reply:
Yeah, but its doesn't fit logically.
Its not part of a "game install".

I think it would fit better to pass the launch options directly to Rust, either via a string or an array (a singular string would cause problems with flags that have quoted values)

Reply:
Ah, misunderstood you there.